packages/tempus/src/Tempus_StepperNewmarkExplicitAForm_impl.hpp

```diff
@@ -196,7 +196,9 @@ void StepperNewmarkExplicitAForm<Scalar>::setInitialConditions(
     auto f       = initialState->getX()->clone_v();
     this->evaluateExplicitODE(f, x, xDot, initialState->getTime());
     Thyra::Vp_StV(f.ptr(), Scalar(-1.0), *(xDotDot));
-    Scalar reldiff = Thyra::norm(*f)/Thyra::norm(*xDotDot);
+    Scalar reldiff = Thyra::norm(*f);
+    //The following logic is to prevent FPEs  
+    if (Thyra::norm(*xDotDot) > 1.0e-12) reldiff /= Thyra::norm(*xDotDot); 
```

Comment:
So I like the compare, but we should probably used a relative compare 
```
if (Thyra::norm(*x) > eps*reldiff) reldiff /= Thyra::norm(*x); 
```
Also I tried to be consistent and use x instead of xDotDot across the Steppers.  that means the output below this needs changing.